mindee/documents/invoice_splitter/invoice_splitter_v1.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any, Dict, List, Optional, TypeVar
+
+from mindee.documents.base import Document, TypeApiPrediction, clean_out_string
+
+
+class PageGroup:
+    """Page Group class for Invoice splitter."""
+
+    page_indexes: List[int] = []
+    """Index of each page"""
+    confidence: float = 0.0
+    """Confidence score"""
+
+    def __init__(self, prediction: Dict[str, Any]):
+        self.page_indexes = prediction["page_indexes"]
+        try:
+            self.confidence = float(prediction["confidence"])
+        except (KeyError, TypeError):
+            pass
+
+    def __str__(self) -> str:
+        return f"page indexes: {', '.join([str(page_index) for page_index in self.page_indexes])}"
+
+
+class InvoiceSplitterV1(Document):
+    """
+    Invoice Splitter prediction results.
+
+    Currently uses the API's async endpoints.
+    """
```

Comment:
```suggestion
    """Invoice Splitter v1 prediction results."""
```
The prediction results are independent of the way the API is called.